addons/point_of_sale/static/src/xml/Chrome.xml

```diff
@@ -4,19 +4,21 @@
     <t t-name="Chrome" owl="1">
         <div class="pos" t-att-class="{ 'big-scrollbars': state.hasBigScrollBars }">
             <div class="pos-receipt-print"></div>
-            <div class="pos-topheader" t-att-class="{ oe_hidden: state.uiState !== 'READY' }">
+            <div class="pos-topheader position-absolute start-0 top-0 d-flex w-100 m-0 p-0 fs-5 text-gray" t-att-class="{ oe_hidden: state.uiState !== 'READY' }">
```

Comment:
Hello, as we are refactoring the css, couldn't we just put the flex rules in the scss file corresponding to keep clear xml files and not add noize with class only there for style?

Reply:
I am not sure position-absolute is needed